plugins.json

```diff
@@ -16,9 +165,9 @@
       "description": "GitLab plugin for Mattermost.",
       "homepage_url": "https://github.com/mattermost/mattermost-plugin-gitlab",
       "support_url": "https://github.com/mattermost/mattermost-plugin-gitlab/issues",
-      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.1",
```

Comment:
I think these changes should be reverted, to keep the `1.8.0` release entry in tact

Reply:
@mickmister Reverted